app/src/main/java/com/ataulm/whatsnext/film/PeopleWidget.kt

```diff
@@ -26,7 +26,7 @@ class PeopleWidget constructor(context: Context, attrs: AttributeSet) : LinearLa
     private fun extractCustomAttributes(context: Context, attrs: AttributeSet): String {
         val typedArray = context.obtainStyledAttributes(attrs, R.styleable.PeopleWidget)
         try {
-            return typedArray.getString(R.styleable.PeopleWidget_label)
+            return typedArray.getString(R.styleable.PeopleWidget_label)!!
```

Comment:
I have no idea why this suddenly started to complain. The kotlin version was updated, but not sure why that would trigger this to report that it's nullable.